resources/test/valid/chainspec.toml

```diff
@@ -10,6 +10,7 @@ accounts_path = 'accounts.csv'
 validator_slots = 5
 auction_delay = 3
 locked_funds_period = 0
+round_seigniorage_rate = [6_414, 623_437_335_209]
```

Comment:
…and here it's 13. (I guess it doesn't matter in the tests; but maybe we can just set it to 14?)